diff --git a/src/invidious/helpers/errors.cr b/src/invidious/helpers/errors.cr
index b2df682d..7534ef31 100644
--- a/src/invidious/helpers/errors.cr
+++ b/src/invidious/helpers/errors.cr
@@ -84,7 +84,12 @@ def error_template_helper(env : HTTP::Server::Context, status_code : Int32, mess
 
   locale = env.get("preferences").as(Preferences).locale
 
-  error_message = translate(locale, message)
+  if message == "This helps protect our community. Learn more"
+    error_message = "<h2>Instance is blocked temporarily</h2><p>This instance uses proxies to bypass blockage from YouTube.<br/>Right now, all of those proxies are blocked.</p><p>Please <b>wait for up to two minutes</b> until the automated background script has sucessfully rotated the proxies and refresh this site afterwards.</p><p>Please don't spam the refresh button, this won't make the process faster.</p><p><b>Please don't report this issue on github, we already know.</b></p>"
+  else
+    error_message = translate(locale, message)
+  end
+
   next_steps = error_redirect_helper(env)
 
   return templated "error"
